scripts/tasks/src/metadata-utils.ts

```diff
@@ -0,0 +1,55 @@
+import * as fs from 'node:fs';
+import * as path from 'node:path';
+
+import { isConvergedPackage, workspaceRoot } from '@fluentui/scripts-monorepo';
+import * as glob from 'glob';
+
+export function getRawMetadata(projectRoot: string) {
+  const root = path.resolve(workspaceRoot, projectRoot);
+  const packageJsonPath = path.join(root, 'package.json');
```

Comment:
avoiding nx readProjectConfiguration makes this 99% faster. Also we don't need all the under the hood check that nx api does for all our tasks.